src/classes/ADDR_Account_TDTM.cls

```diff
@@ -48,123 +51,122 @@ public class ADDR_Account_TDTM extends TDTM_Runnable {
         TDTM_Runnable.Action triggerAction, Schema.DescribeSObjectResult objResult) {
             
         DmlWrapper dmlWrapper = new DmlWrapper();  
-        list<Account> listAccCreateAddr = new list<Account>();      
+        List<Account> ListAccCreateAddr = new List<Account>();      
+        List<Account> ListAccsAddrInfoCleared = new List<Account>();
        
-        integer i = -1;        
+        Integer i = -1;        
```

Comment:
This variable doesn't seem to have a very insightful name. Please give variables names that suggest their purpose.